framework/test/mocha/unit/modules/chain/chain.js

```diff
@@ -329,4 +334,117 @@ describe('Chain', () => {
 			return expect(stubs.modules.module2.cleanup).to.have.been.called;
 		});
 	});
+
+	describe('#_startForging', () => {
```

Comment:
Please always use `// Arrange & Act & Assert` comments as much as possible. It will also help us to organize our code in better way.